.github/workflows/call-plugin-workflow.yml

```diff
@@ -0,0 +1,66 @@
+name: Run plugin workflow
+
+on:
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - main
+
+jobs:
+    call-test-workflow:
+        name: Call workflow for tests
+        strategy:
+            matrix:
+                include:
+                    - shopware-version: '5.6'
+                      php-version: '7.2'
+                      mysql-version: '5.7'
+                    - shopware-version: '5.7'
+                      php-version: '7.4'
+                      mysql-version: '8.0'
+                    - shopware-version: '5.7'
+                      php-version: '8.2'
+                      mysql-version: '8.0'
+
+        uses: shopware5/docker-images-testing/.github/workflows/php-unit-tests-shopware.yml@main
+        with:
+            plugin-name: SwagUserPrice
+            php-version: ${{ matrix.php-version }}
+            shopware-version: ${{ matrix.shopware-version }}
+            mysql-version: ${{ matrix.mysql-version }}
+
+    call-analyse-workflow:
+        name: Analyse code for SwagUserPrice
+        uses: shopware5/docker-images-testing/.github/workflows/php-code-analysis.yml@main
+        with:
+            plugin-name: SwagUserPrice
+
+    javascript-code-analysis:
```

Comment:
please add this plugin to this ticket https://shopware.atlassian.net/browse/PT-13120

would be nice if you could also copy the code there. we should make that a reusable workflow, too

Reply:
Done and deleted :+1: